editor/src/core/workers/parser-printer/parser-printer-uids.spec.tsx

```diff
@@ -113,7 +113,7 @@ describe('parseCode', () => {
     )
 
     const result = getAllUniqueUids(projectContents)
-    expect(result.uniqueIDs).toHaveLength(8)
+    expect(result.uniqueIDs).toHaveLength(7)
```

Comment:
is this number going down because of the reduced number of empty arbitrary blocks?

Reply:
Yeah, these are the arbitrary blocks that used to be a handful of spaces and a line feed.